### PR TITLE
[AAE-11496] Make aca-shared buildable

### PR DESCRIPTION
**Pull request by BSekula**
**Please check if the PR fulfills these requirements**

> - [ ] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [x] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

**What is the new behaviour?**

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: projects/aca-shared/src/lib/components/document-base-page/document-base-page.component.ts [hunk 1]
```diff
@@ -31,7 +31,7 @@ import { Store } from '@ngrx/store';
 import { MinimalNodeEntity, MinimalNodeEntryEntity, NodePaging } from '@alfresco/js-api';
```
Comment (BSekula) [inline projects/aca-shared/src/lib/components/document-base-page/document-base-page.component.ts:31]: This file is moved from `aca-content/../page.components.ts` 

It is going to be used in both libraries, `aca-content` and `aca-preview` therefore I moved it to `shared` lib (so `aca-preview` does not depends on `aca-content`) 

ps. aca-preview is going to be creted [here](https://github.com/Alfresco/alfresco-content-app/pull/2859/files)

File: projects/aca-shared/store/src/public-api.ts [hunk 2]
```diff
@@ -53,3 +53,5 @@ export * from './selectors/app.selectors';
 export * from './states/app.state';
 
 export * from './store.module';
+
+export * from './testing/initial-app-state.mock';
```
Comment (mauriziovitale) [inline projects/aca-shared/store/src/public-api.ts:57]: we don't export mock otherwise will became part of the bundle

**Review by mauriziovitale — APPROVED** [review]
Please do not export testing mock

File: projects/aca-shared/store/src/public-api.ts (hunk 2 above)
Comment (BSekula) [reply projects/aca-shared/store/src/public-api.ts]: Fixed

**Pull request merged**